webpack.config.production.js

```diff
@@ -1,20 +0,0 @@
-'use strict';
-
-var webpack = require('webpack');
-var baseConfig = require('./webpack.config.base');
-
-var config = Object.create(baseConfig);
-config.plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
```

Comment:
We are setting NODE_ENV for Webpack itself (the Node part). But it won't be polyfilled in generated JS code (the client side) unless we tell webpack a la

```js
new webpack.DefinePlugin({
  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
})
```